frontend/packages/console-shared/src/selectors/common.ts

```diff
@@ -1,10 +1,14 @@
 import * as _ from 'lodash';
-import { K8sResourceCommon } from '@console/internal/module/k8s';
+import { isCSRResource } from '@console/app/src/components/nodes/csr';
+import { K8sResourceCommon, NodeKind } from '@console/internal/module/k8s';
 
 export const hasLabel = (obj: K8sResourceCommon, label: string): boolean =>
   _.has(obj, ['metadata', 'labels', label]);
 export const getName = <A extends K8sResourceCommon = K8sResourceCommon>(value: A) =>
   _.get(value, 'metadata.name') as K8sResourceCommon['metadata']['name'];
+export const getNodeArchitecture = (node: NodeKind) => {
+  return isCSRResource(node) ? '' : node?.status?.nodeInfo?.architecture;
```

Comment:
I'm still uncertain here. How does CSR relate to architecture? Do we need this special case at all? I'd expect all nodes to have a CPU architecture.

@rawagner Are you the right person to ask? :)

Reply:
Thanks for providing the suggestions, modified the code . PTAL.